ui/src/components/erc/ERC.js

```diff
@@ -244,11 +277,21 @@ class ERC extends React.Component {
     }
 
     handleAlertClose() {
-        console.log(this)
+        if(this.props.ojsView){
+            window.open("https://o2r.uni-muenster.de", "_blank")
```

Comment:
Should this really by a constant, or not something defined by the OJS plug-in?

Reply:
Sorry, I don't understand the question! Do you mean there are many more places in the code where this is configured?

Maybe you can explain briefly what exactly happens here - this is not the API endpoint, right? When is this link opened?

Reply:
Ich erkläre es mal in Deutsch, ist glaube ich einfacher:
Wenn ein ERC nicht gefunden wird, also eine ID angegeben wird, die nicht existiert öffnet sich ein Fenster, wo der Nutzer auf dieses Problem hingewiesen wird. In dieser Funktion wird festgelegt was passieren soll, wenn der Nutzer das Fenster schließt.   Ich dachte er soll auf die Hauptseite verwiesen werden, wo alle ERCs angezeigt werden, wenn er sich im OJS View befindet. Was sind deine Ideen?

Reply:
Im OJS View sollte er wieder auf der Journal-Startseite landen.

Reply:
Ok, dann Frage ich mal Tom, wie die URL zur Journal-Startseite aussieht

Reply:
Entscheidung: wenn die UI innerhalb von OJS ist, dann kann man die Fehlermeldung nicht schließen.